metadata_tools/column/COLUMNS_RING.py

```
@@ -92,6 +92,8 @@
     (("body_diameter_in_pixels", planet_ring,
                             util.replacement_fn("defs.RING_SYSTEM_RADII", defs.BODYX)),
                                                             ("",   "",  "")),
+    (("pole_clock_angle",       defs.BODYX),                    ("",   "",  "")),
```

Ruff (0.14.10)

Undefined name `defs`

(F821)